wasm/demo/src/main.js

```diff
@@ -72,3 +73,55 @@ snippets.addEventListener('change', updateSnippet);
 // Run once for demo (updateSnippet b/c the browser might try to keep the same
 // option selected for the `select`, but the textarea won't be updated)
 updateSnippet();
+
+const prompt = ">>>>> ";
+
+const term = new Terminal();
+term.open(document.getElementById('terminal'));
+term.write(prompt);
+
+function remove_non_ascii(str) {
```

Comment:
We've been using snakeCase names for JS stuff, so if you want you could keep that naming style here.